preview/upgrade/autoosupgrade-doc.md

```diff
@@ -48,6 +48,10 @@ The following SKUs are currently supported (more will be added):
 | MicrosoftWindowsServer  | WindowsServer | 2012-R2-Datacenter | latest   |
 |  MicrosoftWindowsServer | WindowsServer | 2016-Datacenter    | latest   |
 | Canonical               | UbuntuServer  | 16.04-LTS          | latest  |
+|MicrosoftCblMariner      | Cbl-Mariner   | cbl-mariner-1      | latest   |
```

Comment:
I cannot comment in the unchanged text, so commenting here for line 42 - there is a typo for "proerty" which should be property.